physics/docs/pdftxt/CLM_LAKE.txt

```diff
@@ -0,0 +1,59 @@
+and shortwave and longwave radiative fluxes. The CLM lake model then provides latent heat and sensible heat fluxes back to the 
+atmosphere. It also computes 2-m temperature/moisture, skin temperature, lake temperature, ice fraction, ice thickness, snow water
+equivalent and snow depth. The CLM lake model divides the vertical lake profile into 10 layers driven by wind-driven eddies. The
+thickness of the top layer is fixed to 10-cm and the rest of the lake depth is divided evenly into the other 9 layers. Energy
+transfer (heat and kinetic energy) occurs between lake layers via eddy and molecular diffusion as a function of the vertical 
+temperature gradient. The CLM lake model also uses a 10-layer soil model beneath the lake, a multi-layer ice formation model and
+up to 5-layer snow-on-ice model. Multiple layers in lake model have the potential to better represent vertical mixing processes 
+in the lake.
+
+Testing of the CLM lake model within RAP/HRRR applications showed computational efficiency of the model with no change of even
+0.1% in run time. The lake/snow variables have to be continuously transfered within the CLM lake model from one forecast to another,
+constrained by the atmospheric data assimilation. The lake-cycling initialization in RAP/HRRR has been effective overall, owing to 
+accurate houly estimates of near-surface temperature, moisture and winds, and shortwave and longwave estimates provided to the 1-d CLM
+lake model every time step (Benjamin et al. (2022) \cite gmd-15-6659-2022). Cycling techniques showed improvements over initializing
+lake temperatures from the SST analysis, which is problematic for small water bodies. The improvements are particularly eminent during transition 
+periods between cold and warm seasons, and in the regions with anomalous weather conditions. The CLM lake model has the potential 
+to improve surface prediction in the vicinity of small lakes.
+
+The CLM lake model requires bathymetry for the lake points in the model domain. Grid points are assigned as lake points when the 
+fraction of lake coverage in the grid cell exceeds 50% and when this point is disconnected from oceans. The lake water mask is 
+therefore binary, set to either 1 or 0. This binary approach for models with higher horizontal resolution --- for example, 3-km resolution in 
+in the UFS SRW App --- is capable of capturing the effect of lakes on regional heat and moisture fluxes.
+
+Lake depths for the RRFS lake configuration (Fig.1) are assigned from a global dataset provided by Kourzeneva et al.(2012) \cite Kourzeneva_2012,
+this dataset is referred to as GLOBv3 bathymetry in the UFS_UTL.
+
+@image html https://user-images.githubusercontent.com/12705538/250180794-76af93a2-a7ba-4e9a-9478-5657198862b8.png "Figure 1: Lake depths for lakes in the 3-km RRFS domain." width=600
+
+To cold-start the CLM lake model in the UFS weather model:
```

Comment:
I changed this line from "UFS SRW App" to "UFS weather model', since the UFS SRW App doesn't have any option to activate the CLM lake model without manually editing the weather model namelist.

Reply:
Tanya's original input is "To cold-start the CLM lake model in RRFS:" I think regional application/configuration is key here, but I understand Ligia wants to highlight this doc is for SRW App. How about "To cold-start the CLM lake model in regional configurations of the UFS" or something along the lines?

Reply:
sounds good to me.